iModelCore/Units/src/Formatting/NumericFormatSpec.cpp

```diff
@@ -959,4 +963,44 @@ double NumericFormatSpec::RoundDouble(double dval, double roundTo)
     return (dval < 0.0) ? -rnd : rnd;
     }
 
+Utf8String NumericFormatSpec::FormatToFractionalRatio(double dval) const
+{   
+    double reciprocal;
+    if (dval == 0.0)
+        reciprocal = 0.0;
+    else
+        reciprocal = 1.0 / dval;
+        
+    return "1:" + Format(reciprocal);

```

Comment:
Should this be reversed if dval is greater than 1.0? The example you linked on the github issue says 0.2857 should turn into "1:3.5" but how about a value greater than 1? I believe in that case the second number should be 1 and the first should be variable.